### DISCUSSION TITLE
一个产品有相关的多个链接 怎么写？

### DISCUSSION THREAD

**Discussion by lgh06** [xcatliu/awesome-aigc#6] [General]
比如AltDiffusion， 一个是公司展示页面，一个是huggingface模型页面，另一个是Github页面，我建议都写上，因为他们三个确实强相关，但是出品方却没有在醒目位置把这三个页面关联起来，我搜了不少时间才大概搞明白用法…

**Comment by xcatliu** [comment]
可以先都写上吧，后面可以做成结构化的 json 或 yaml 维护
